network-api/networkapi/settings.py

```diff
@@ -412,7 +412,7 @@
 LANGUAGES = (
     ('en', 'English'),
     ('de', 'German'),
-    ('pt', 'Portuguese'),
+    ('pt-BR', 'Portuguese (Brazil)'),
```

Comment:
This is the only difference from the old PR change: https://github.com/mozilla/foundation.mozilla.org/pull/6193/files#diff-e09a0533a78bcb9c2263afabeb95633572db893f9ddcd40d883332a5015e2362

If it's set to `('pt', 'Portuguese (Brazil)'),`, we'll get status 500 back when accessing `/pt-BR` urls.

Reply:
right, `pt-BR` is what we want here